internal/cluster/client_message_test.go

```diff
@@ -0,0 +1,152 @@
+package cluster
+
+import (
+	"bytes"
+	"github.com/hazelcast/hazelcast-go-client/internal/proto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"testing"
+)
+
+func TestClientMessageReader(t *testing.T) {
+	testCases := []struct {
+		f    func(t *testing.T)
+		name string
+	}{
+		{name: "readSingleFrame", f: readSingleFrame},
+		{name: "readMultiFrameMessage", f: readMultiFrameMessage},
+		{name: "readFramesInMultipleCallsToRead", f: readFramesInMultipleCallsToRead},
+		{name: "readWhenTheFrameLengthAndFlagsNotReceivedAtFirst", f: readWhenTheFrameLengthAndFlagsNotReceivedAtFirst},
+		{name: "readFramesInMultipleCallsToReadFromWhenLastPieceIsSmall", f: readFramesInMultipleCallsToReadFromWhenLastPieceIsSmall},
+	}
+	for _, tc := range testCases {
+		tc := tc
```

Comment:
Those tests are not running concurrently, we don't need to copy test cases.

Reply:
Removed at https://github.com/hazelcast/hazelcast-go-client/pull/863/commits/73051a7840e4f303b95e8fd0abc090403aff37b6.